common/base64.h

```diff
@@ -0,0 +1,10 @@
+#ifndef _BASE64_H
+#define _BASE64_H
+
+extern int __b64_ntop(u_char const *src, size_t srclength, char *target, size_t targsize);
```

Comment:
for a library function like this, it might be nice to use the common library prefix rfb so these should be called sth like rfbBase64NtoP()